### ISSUE TITLE
Temperature reading update

### ISSUE THREAD

**Issue by polo78** [MarlinFirmware/Marlin#3645]
Hi,
I would like to know how frequently the temperature is read by marlin.
Thanks

**Comment by thinkyhead** [comment]
I believe the temperature ISR `ISR(TIMER0_COMPB_vect)` runs at ~125Hz because:

``` cpp
  OCR0B = 128;
  SBI(TIMSK0, OCIE0B);
```

(I may be wrong – Arduino's timer documentation is hard to follow. So someone may correct me.)

One temperature sensor is read on every other interrupt, in order. So if you have 2 temperature sensors, they will each be read every 4 interrupts, or about 30 times per second. For each ADC sensor you add, it delays reading by 2 more interrupts.

**Comment by Blue-Marlin** [comment]
Then when OVERSAMPLENR (16) samples are made current_temperature is updated -> ~2 times / second.
